js-intro.js

```diff
@@ -2,61 +2,105 @@
 
 // Consider this variable:
 
-var mantra = "Be the dev";
+//var mantra = "B"
 
 // 1. Write the code that returns 'B' from mantra.
+   // console.log(mantra)
+
 
 // 2. Write the code that determines if there is a 'x' in mantra.
 
+//var mantra = "Be the Dev"
+//function mantra (){
+  // if(mantra.includes("")){
+  //      return "true"
+//    } else {
+//        return "false"
+//    }
+//}
+  //  console.log(mantra.includes("x"))
+    
+
 // 3. Write the code that determines if there is a 'v' in mantra.
+//var mantra = "Be the Dev"
+//function mantra (){
+ //   if(mantra.includes("")){
+  //      return "true"
+ //   } else {
+ //       return "false"
+ //   }
+//}
+ //   console.log(mantra.includes("v"))
 
 // Stretch: Write the code that returns the position of 'v' in mantra.
 
+//var mantra = "Be the Dev"
+//function mantra (){
+  //  return(mantra.indexOf(""))
+  //  }
+
+ //   console.log(mantra.indexOf("v"))
+
 
 // Consider the following statement:
 
-var codingIsEasy
-var imBored
+//var codingIsEasy
```

Comment:
For this and the next exercise, assign a boolean value to the variables for the right block to run